src/daemon/command.c

```diff
@@ -136,7 +137,7 @@ int readcmd(usbdevice* kb, char* line){
             continue;
         }
         // Reject anything not related to fwupdate if device has a bricked FW
-        if(NEEDS_FW_UPDATE(kb) && command != FWUPDATE && command != NOTIFYON && command != NOTIFYOFF && command != RESET)
+        if(NEEDS_FW_UPDATE(kb) && command != FWUPDATE && command != NOTIFYON && command != NOTIFYOFF && command != RESET && command != PAIR)
```

Comment:
As per our discussion, this isn't needed, as we can't pair devices in bootloader mode anyway.